tests/x11/libqt5_qtbase.pm

```diff
@@ -21,6 +21,13 @@ use version_utils 'is_sle';
 use registration qw(cleanup_registration register_product add_suseconnect_product get_addon_fullname remove_suseconnect_product);
 
 sub run {
+    if (is_sle) {
+        select_console 'root-console';
+        # Activating development-tools module to install libqt5-qttools package
+        register_product();
```

Comment:
is this function needed?

Reply:
hmm.. I think I just assumed this way by referring to some other test modules, I will check whether it is required before calling `add_suseconnect_product()` function.
Thanks for reviewing!

Reply:
Anna, Thanks for pointing this out, calling `register_product` is not needed for activating the module. I have pushed the new commit, please review.